Pyrado/pyrado/utils/input_output.py

```diff
@@ -27,15 +27,18 @@
 # POSSIBILITY OF SUCH DAMAGE.
 
 import contextlib
-import numpy as np
 import select
 import sys
-from colorama import Fore, Style
-from typing import Sequence, Iterable, Callable, Any
+import typing
+from typing import Sequence, Iterable, Callable
 
+import numpy as np
 import pyrado
+from colorama import Fore, Style
 from pyrado.utils import run_once
 
+T = typing.TypeVar("T")
```

Comment:
is this necessary, can't We simply use Any?

Reply:
Using the `TypeVar` has the advantage that it is known to e.g. PyCharm what the return type will be (in the case of asking for an experiment, PyCharm will know that `select_query` will return an `Experiment`).

Reply:
I see, then we keep it